docs/samples/Microsoft.ML.Samples/Static/LightGBMMulticlassWithInMemoryData.cs

```diff
@@ -0,0 +1,163 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.LightGBM.StaticPipe;
+using Microsoft.ML.StaticPipe;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.ML.Samples.Static
+{
+    class LightGBMMulticlassWithInMemoryData
```

Comment:
>LightGBMMulticlassWithInMemoryData [](start = 10, length = 34)

this needs to be referenced from some other file, otherwise it won't display in the documentation. 

Maybe from the LightGBMStatics catalog, if we have one already?

Reply:
I am not quite following. Do you have any example? In addition, could we reference a sample from our main library? I thought sample project and ML.NET are independent?

---
In reply to: [244876344](https://github.com/dotnet/machinelearning/pull/1953#discussion_r244876344) [](ancestors = 244876344)